tests/GatewayTest.php

```diff
@@ -18,15 +18,15 @@ public function testPurchase()
         $request = $this->gateway->purchase(array('amount' => '10.00'));
 
         $this->assertInstanceOf('Omnipay\PayZen\Message\PurchaseRequest', $request);
-        $this->assertSame('1000', $request->getAmount());
+        $this->assertSame('10.00', $request->getAmount());
```

Comment:
Hmm wait, why did you change that ?

Reply:
because it seems that in Omnipay v3, the format of amount changed. But it does not impact the format of amount which is used when redirected to Payzen page: I still have i.e `"vads_amount" => "5500"` for 55€.